src/views/homepage/components/chiclets/index.tsx

```diff
@@ -35,7 +35,7 @@ const Chiclets = () => {
 									<a className={s.navListLink} href={href}>
 										<div className={s.navListLinkContent}>
 											<ProductIcon productSlug={productSlug} size={24} />
-											{text}
+											<span className={s.chicletText}>{text}</span>
```

Comment:
🗒️ This text needed to be wrapped to apply styles to it. Didn't notice before that styles hadn't been applied already. 😊